server/documents/document.js

```diff
@@ -229,7 +229,7 @@ Meteor.publish('Document.one', function documentOne(args) {
     else {
       return Document.documents.find({
         _id: args.documentId,
-        visibility: {$ne: Document.VISIBILITY_LEVELS.PRIVATE},
+        publishedAt: {$ne: null},
```

Comment:
Why this change? We had quite a bit of discussion about this. I think we can have this as it was before, that public or published documents can be seen by anyone. Appcivist will then control in more detail this for now and for Appcivist there will be only public or private control, no "listed" yet, so I think it works out.

The issue is that we do not have a concept of groups in PeerDoc. And production instance of PeerDoc is shared between multiple sets of users of Appcivist. So locking down to all registered users does not mean just to one set of users of Appcivist. So I would prefer that we have it that public is public and Appcivist then controls groups.